build.gradle

```diff
@@ -706,7 +706,6 @@ project(':iceberg-hive-metastore') {
       exclude group: 'com.google.code.findbugs', module: 'jsr305'
       exclude group: 'org.eclipse.jetty.aggregate', module: 'jetty-all'
       exclude group: 'org.eclipse.jetty.orbit', module: 'javax.servlet'
-      exclude group: 'org.apache.parquet', module: 'parquet-hadoop-bundle'
```

Comment:
Not sure if change in intentional. Doesn't seem to be relevant to the PR for me.

Reply:
Without this, it does not compile...
> Task :iceberg-hive-metastore:compileJava FAILED
HMSTablePropertyHelper.java:46: error: package org.apache.parquet.hadoop does not exist
import org.apache.parquet.hadoop.ParquetOutputFormat;

Reply:
Happily this one is not needed, since the `ParquetOutputFormat` is not needed

Reply:
Reverted.